src/widgets/settingspages/GeneralPage.cpp

```diff
@@ -1029,6 +1029,10 @@ void GeneralPage::initLayout(GeneralPageView &layout)
                        false,
                        "Make all clickable links lowercase to deter "
                        "phishing attempts.");
+    layout.addCheckbox(
+        "Show user's pronouns in user card", s.showPronouns, false,
+        "Shows users' pronouns in their user card. "
+        "Pronouns are retrieved from alejo.io when the user card is opened.");
```

Comment:
If possible I'd like to add information about the pronoun API's privacy policy here. I've asked them if they have one and will request one be added here if they do.